client/src/boundary/interactor/GetBoardInteractor.test.ts

```diff
@@ -0,0 +1,30 @@
+import { GetBoardInteractor } from "./GetBoardInteractor";
+import BoundaryBoard from "boundary/model/BoundaryBoard";
+import { mock, MockProxy } from "jest-mock-extended";
+import GameStorage from "services/api/GameStorage";
+import Board from "domain/Board";
+import { GameD2BConverter } from "./GameD2BConverter";
+
+describe("Get board interactor", () => {
+  let gameStorage: MockProxy<GameStorage>;
+  let gameD2BConverter: MockProxy<GameD2BConverter>;
+  let getBoardInteractor: GetBoardInteractor;
+
+  beforeEach(() => {
+    gameStorage = mock<GameStorage>();
+    gameD2BConverter = mock<GameD2BConverter>();
+    getBoardInteractor = new GetBoardInteractor(gameStorage, gameD2BConverter);
+  });
+
+  it("Board is returned", () => {
```

Comment:
turėtų gautis skaitomas angliškas sakinys perskaičius (describe Get board interactor it _returns board_) jei naudoji "it", o jei taip vadini, tai naudok "test" ne "it" :D